### PR TITLE
[Element 2.1] Enhancement terminal report

### PR DESCRIPTION
**Pull request by phileba**
resolve https://tricentisflood.atlassian.net/browse/FLO-2040

### CODE REVIEW

File: packages/core/src/interface/ITest.ts [hunk 1]
```diff
@@ -35,7 +35,6 @@ export interface ITest {
 		testDataRecord: any,
 	): Promise<void>
 	doStepDelay(): Promise<void>
-	willRunCommand(testObserver: TestObserver, browser: Browser<Step>, command: string): Promise<void>
```
Comment (honglax) [inline packages/core/src/interface/ITest.ts:38]: Why did you remove `willRunCommand()`?

IMO, it should be changed to 
```
willRunCommand(
    testObserver: TestObserver,
    browser: Browser<Step>,
    command: string,
    args: string,
): Promise<void>
```

File: packages/core/src/runtime/Test.ts [hunk 2]
```diff
@@ -255,6 +257,12 @@ export default class Test implements ITest {
 				}
 			})
 			await this.afterRunSteps(stepIterator)
+			if (this.settings.showScreenshot && process.env.TERM_PROGRAM === 'iTerm.app') {
+				for (const path of browser.fetchScreenshots()) {
+					console.log(basename(path))
```
Comment (honglax) [inline packages/core/src/runtime/Test.ts:262]: Do we need to keep this `console.log()`

File: packages/core/src/interface/ITest.ts (hunk 1 above)
Comment (phileba) [reply packages/core/src/interface/ITest.ts:38]: okay, it's my fault lol

File: packages/core/src/runtime/Test.ts (hunk 2 above)
Comment (phileba) [reply packages/core/src/runtime/Test.ts:262]: yes, we need to show the file name of the screenshot

**Review by honglax — APPROVED** [review]
